pkgs/applications/editors/emacs/elisp-packages/cask/default.nix

```diff
@@ -30,27 +54,27 @@ stdenv.mkDerivation rec {
     runHook preInstall
 
     mkdir -p $out/bin
-    dir=$out/share/emacs/site-lisp/cask
-    install -Dm444 -t $dir     *.el *.elc
-    install -Dm555 -t $dir/bin bin/cask
-    touch $out/.no-upgrade
-    ln -s $dir/bin/cask $out/bin/cask
+    install -Dm444 -t $out/share/emacs/site-lisp/cask *.el *.elc
+    install -Dm555 -t $out/share/emacs/site-lisp/cask/bin bin/cask
+    ln -s $out/share/emacs/site-lisp/cask/bin/cask $out/bin/cask
 
     runHook postInstall
   '';
 
+  postFixup = ''
+    touch $out/.no-upgrade
```

Comment:
What is this for, can you add a comment?

Reply:
It came from the old expression, and it was used to avoid auto-upgrade.

Now I am feeling inclined to remove it for good.